### PR TITLE
new(scripts): add bottlerocket support in falco-driver-loader.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

Adds support for bottlerocket OS in falco-driver-loader script.
See relevant PRs:
* https://github.com/falcosecurity/kernel-crawler/pull/79
* https://github.com/falcosecurity/driverkit/pull/239

Kernel-crawler will output a json similar to the minikube one:
```
{
"kernelversion": "1_1.11.1-vmware",
"kernelrelease": "5.15.59",
"target": "bottlerocket",
"kernelconfigdata": "..."
}
```

Basically, `kernelversion` ships OS version (`_1.11.1`) and flavor (`-vmware`).
Therefore, the changes in falco-driver-loader.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
new(scripts): add bottlerocket support in falco-driver-loader
```

### CODE REVIEW

**Comment by FedeDP** [comment]
Wait on https://github.com/falcosecurity/kernel-crawler/pull/79 to be merged first.

**Comment by FedeDP** [comment]
Kernel-crawler PR was merged.

**Comment by FedeDP** [comment]
We now have bottlerocket entries in kernel-crawler generated json: https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=BottleRocket.

I think this is safe to be merged ;)
/milestone 0.34.0

**Review by jasondellaluce — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]
